src/commands/start.ts

```diff
@@ -194,6 +209,25 @@ export default class Start extends Command {
     }
   }
 
+  private async listSchemas() {
+    CliUx.ux.info(this.getStatus())
+    await ListSchemas.run(['-w'])
+    this.breadcrumbs.push(showSchemas)
+  }
+
+  private async createSchema() {
+    CliUx.ux.info(this.getStatus())
+    await CreateSchema.run([
+      '-s',
+      `${await schemaJSONFilePath()}`,
+      `-p`,
+      `${await schemaPublicPrivate()}`,
+      '-d',
+      `${await schemaDescription()}`,
+    ])
+    this.breadcrumbs.push(createSchema)
+  }
+
```

Comment:
Isn't this wizard start file getting quite big now ?
Does it make sens to create a folder and to extract the different functions into their own scope files ?

Reply:
I'll try to continue working on this on the refactor branch